src/jdk.compiler/share/classes/com/sun/tools/javac/processing/PrintingProcessor.java

```diff
@@ -126,7 +126,7 @@ public PrintingElementVisitor visitExecutable(ExecutableElement e, Boolean p) {
                 kind != INSTANCE_INIT) {
                 Element enclosing = e.getEnclosingElement();
 
-                // Don't print out the constructor of an anonymous or unnamed class
+                // Don't print out the constructor of an anonymous
```

Comment:
Please add back "class" so that the comment reads "Don't print out the constructor of an anonymous class."